script/prerequisitesForGetToolchains.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I see the change in shebang. So, do we have to change `sh` to `bash` in `prerequistesForGetToolchains` too?

Reply:
Our project usually uses bash shell. So I changed it too. :)

Reply:
I see :) Thank you!